platformics/codegen/tests/test_schemas/platformics.yaml

```diff
@@ -346,6 +346,24 @@ classes:
     annotations:
       plural: UpstreamDatabases
 
+  BulkDownload:
+    is_a: Entity
+    mixins:
+      - EntityMixin
+    attributes:
+      download_display_name:
+        range: string
+        required: true
+        annotations:
+          mutable: false
+      file:
+        range: File
+        readonly: true
+        annotations:
+          cascade_delete: true
+    annotations:
+      plural: BulkDownloads
```

Comment:
Is there a TODO or ticket somewhere to add tests for the BulkDownload delete?

Reply:
Good point, added it as a todo on the https://czi-tech.atlassian.net/browse/CZID-9412 ticket